pkgs/development/compilers/llvm/11/llvm.nix

```diff
@@ -35,6 +35,8 @@ in stdenv.mkDerivation (rec {
   src = fetch pname "0a5mb65xa5bal8q6cb37xgkqis2bip87fsafgq3wbsva9cjprn6c";
   polly_src = fetch "polly" "1smrqm9s0r2g9h0v0nil6y9wn2ih4l5bddk4dhgn538ngc7cxpq8";
 
+  patches = [ ./llvm-genode.patch ];
```

Comment:
Any links to upstream discussions of Genode support?

Reply:
Not yet, there is someone else working on porting the LLVM runtime libraries to Genode, and they want to wait until that is complete. If that doesn't happen before 12, then I don't have a problem with leaving that release unpatched.

Reply:
Hm. A link to the upstream discussion of that porting work, and then whatever value they use in their own tree, is probably also fine. Just to understand where we are standing relative to the actual LLVM community.